tst/testinstall/trace.tst

```diff
@@ -46,4 +46,16 @@ gap> Size(a);
 gap> TraceImmediateMethods(false);
 
 #
```

Comment:
These tests don't cover all of the methods that can be traced. I am happy with leaving the tests as they are now though, I am just not sure if we have a policy for these situations?

Reply:
I will add some more tests

Reply:
I've now added a bunch more tests. In many cases the output may be surprising, but that seems to be what GAP uses internally, there isn't a simple mapping to the underlying methods (which does make this function harder to use, I'll admit.

Reply:
Thanks!